lib/src/test/java/io/ably/lib/test/realtime/RealtimePresenceTest.java

```diff
@@ -302,6 +303,10 @@ public void enter_leave_simple() {
 			leaveComplete.waitFor();
 			assertTrue("Verify leave callback called on completion", leaveComplete.success);
 
+			try {
+				assertEquals("Verify item is removed from the presence map", client1Channel.presence.get(testClientId1, false).length, 0);
```

Comment:
Why not test that the member has gone (ie based on `memberKey` instead of `clientId` ?

Edit: I guess you can't because it's private